molecule/openstack/create.yml

```diff
@@ -62,8 +62,8 @@
     - name: Create molecule instance(s)
       os_server:
         name: "{{ item.name }}"
-        image: "{{ item.image }}"
-        flavor: "{{ item.flavor }}"
+        image: "{{ nova_image }}"
```

Comment:
Where are these values coming from? Are `item.image` and `item.flavor` not working for you? If so, you probably need to update `molecule.yml` to include appropriate values there.

Reply:
Fixed and updated